modules/util/ui/UIState.py

```
@@ -241,12 +244,17 @@ def __create_vars(self, obj):
         return new_vars
 
     def __set_vars(self, obj):
+        if obj is None:
+            return
+
         is_dict = isinstance(obj, dict)
         is_config = isinstance(obj, BaseConfig)
         iterable = obj.items() if is_dict else vars(obj).items()
 
         if is_config:
             for name, var_type in obj.types.items():
+                if name not in self.__vars:
```

Same for both 256-257 and 276-277, set_vars is meant to update existing UI variables, Im fairly sure this would silently skip them.

What scenario necessitated you to add this? I havent seen errors with set vars so far on `main`